### PR TITLE
Use plone resource instead of browser resource

### PR DESCRIPTION
**Pull request by wesleybl**
Plone resource is better suited for the Resource Registry.

### CODE REVIEW

File: src/collective/cover/browser/templates/tilelist.pt [hunk 1]
```diff
@@ -17,7 +17,7 @@
                              data-tile-configurable metadata/is_configurable;
                              title metadata/description;
                              id tile/tile_type">
-            <img src="" tal:attributes="src python:metadata['icon'] if metadata['icon'] else '++resource++collective.cover/img/tile-generic.png';
+            <img src="" tal:attributes="src python:metadata['icon'] if metadata['icon'] else '++plone++collective.cover/img/tile-generic.png';
```
[the remark this answers is not in the document]
Comment (wesleybl) [reply src/collective/cover/browser/templates/tilelist.pt:20]: The `alt` is entered with zpt.

**Comment by mauritsvanrees** [comment]
@fredvd There is a lot of output and GitHub Actions does not show it all by default. In the output below you have line 10, then 11, and then there is a jump to line 990:
```
Run tox -r
10 py36-Plone52 create: /home/runner/work/collective.cover/collective.cover/.tox/py36-Plone52
11 tox: py36-Plone52
990 ___________________________________ summary ____________________________________
991 py36-Plone52: commands succeeded
992 congratulations :)
```

If you click on line 11 the rest of the output is shown, ending with:

```
[989] Total: 273 tests, 0 failures, 0 errors and 0 skipped in 11 minutes 33.804 seconds.
```

So yes, it would be very useful if that extra line of output was included, but at least the tests are run correctly.

**Comment by mauritsvanrees** [comment]
I can't test this PR locally. During the buildout run I get this error:

```
error pixelsmith@2.6.0: The engine "node" is incompatible with this module. Expected version ">= 12.0.0". Got "8.11.2"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
ERROR: npm is known not to run on Node.js v8.11.2
You'll need to upgrade to a newer Node.js version in order to use this
version of npm. You can find the latest version at https://nodejs.org/
```

When seeing this error I made sure to have a recent enough node version with npm:

```
$ nvm use lts/fermium 
$ node --version
v14.19.3
```

Running buildout again gives the same error. It does not seem fatal: buildout seems to finish without error, with exit code zero. Startup fails though, because some files are missing that presumably should have been created by buildout/node/whatever:

```
zope.configuration.exceptions.ConfigurationError: ('No such file', '/Users/maurits/community/collective.cover/src/collective/cover/browser/static/resources.pt')
    File "/Users/maurits/community/collective.cover/parts/instance/etc/site.zcml", line 15.2-15.55
    File "/Users/maurits/community/collective.cover/parts/instance/etc/package-includes/002-collective.cover-configure.zcml", line 1.0-1.60
    File "/Users/maurits/community/collective.cover/src/collective/cover/configure.zcml", line 32.2-32.32
    File "/Users/maurits/community/collective.cover/src/collective/cover/browser/configure.zcml", line 156.2-164.8
```

**Review by mauritsvanrees — APPROVED** [review]
Right, that was it. I *did* throw away stuff beforehand, but not enough. Now it works.

Okay, looks good to me, can be merged.

**Pull request merged**